package.json

```diff
@@ -11,6 +11,9 @@
     "scripts": {
         "build:ts": "tsc",
```

Comment:
Because we don't pre-bundle anymore, I think we can safely now just have simpler scripts like `build`, `buid:watch`.